circuit/pallets/execution-delivery/src/exec_composer/tests.rs

```diff
@@ -110,7 +110,7 @@ pub fn make_compose_out_of_raw_wat_code<T: Config>(
 }
 
 pub fn insert_default_xdns_record() {
-    use pallet_xdns::XdnsRecord;
+    use pallet_xdns::types::XdnsRecord;
```

Comment:
Types vs t3rn-primitives crate? @zannis let's leave in types for now but please extract as a separate issue to align it.

Reply:
Ah I see, yeah I just moved the types to a separate file for clarity, but I can re-export them on the lib.rs so this is aligned with the rest of the codebase.